server.js

```diff
@@ -4380,17 +4380,19 @@ cache(function(data, match, sendBadge, request) {
   };
   var badgeData = getBadgeData(type, data);
   request(options, function(err, res, json) {
-    if (err != null) {
-      badgeData.text[1] = 'inaccessible';
+    if (res && (res.statusCode === 404 || data.state === null)) {
+      badgeData.text[1] = 'not found';
       sendBadge(format, badgeData);
       return;
     }
     try {
       if (type === 'role') {
         badgeData.text[1] = json.namespace + '.' + json.name;
         badgeData.colorscheme = 'blue';
-      } else {
-        badgeData.text[1] = 'unknown';
```

Comment:
This else block was lost and should be restored.

You could add a test to cover it, if you'd like. ^^

Reply:
I couldn't find a way to get there, I think it's dead code but I will add it back if you insist.

I don't know how to test it since I believe that case is already handled a few lines before, see the `if (res && (res.statusCode === 404 || data.state === null)) {` block.

What do you think?

Reply:
Sorry, reading the code again, I think it's not possible to get there since `type` will always be either `role` or `role-downloads` because of the regexp filter. But I'm no expert in this environment.

Reply:
You're totally correct. My bad!

Reply:
no problem thanks